Cargo.toml

```diff
@@ -15,6 +15,12 @@ name = "rusoto"
 readme = "README.md"
 repository = "https://github.com/rusoto/rusoto"
 version = "0.24.0"
+homepage = "https://www.rusoto.org/"
+categories = [ "development-tools" ]
```

Comment:
I'm not sure of this category TBH. I just put it because it seemed to be the category that fit the most with the project. In fact I was looking for something like `library`, `sdk` or `cloud`.

I think we can leave with no category for now and keep an eye if something more appropriately appears.